libs/perun/services/src/lib/auth.service.ts

```diff
@@ -166,10 +51,6 @@ export class AuthService {
     }
```

Comment:
remove from logout method `localStorage.removeItem('refresh_token');` because this is already done by the this.oauthService.logOut();